pkg/process/init.go

```diff
@@ -173,6 +181,57 @@ func (p *Init) Create(ctx context.Context, r *CreateConfig) error {
 	return nil
 }
 
+func (p *Init) validateRuncFeatures(ctx context.Context, bundle string) error {
+	spec, err := readConfig(bundle)
+	if err != nil {
+		return fmt.Errorf("failed to read config: %w", err)
+	}
+	if spec == nil {
```

Comment:
no need to check this because the `readConfig` guarantees it is not nil.

Reply:
Right, thanks!